python-main.js

```diff
@@ -1205,15 +1259,17 @@ function web_editor(config) {
             $("#command-serial").attr("title", serialButton["title-close"]);
             $("#command-serial > .roundlabel").text(serialButton["label-close"]);
 
-            window.daplink.connect()
+            var daplink = usePartialFlashing ? window.dapwrapper.daplink : window.daplink;
+
+            daplink.connect()
             .then( function() {
-                return window.daplink.setSerialBaudrate(115200);
+                return daplink.setSerialBaudrate(115200);
             })
             .then( function() {
-                return window.daplink.getSerialBaudrate();
+                return daplink.getSerialBaudrate();
             })
             .then(function(baud) {
-                window.daplink.startSerialRead(50);
+                daplink.startSerialRead(50);
```

Comment:
We didn't merge this PR to the right branch, can we manually apply this change here? https://github.com/bbcmicrobit/PythonEditor/pull/208

```suggestion
                daplink.startSerialRead(1);
```